docs/src/pages/demos/lists/SimpleList.tsx

```diff
@@ -9,23 +8,23 @@ import Divider from '@material-ui/core/Divider';
 import InboxIcon from '@material-ui/icons/Inbox';
 import DraftsIcon from '@material-ui/icons/Drafts';
 
-const styles = (theme: Theme) =>
+const useStyles = makeStyles((theme: Theme) =>
   createStyles({
```

Comment:
same again

Reply:
Maybe we should remove all of them at once in a different pull request.

Reply:
That would probably be better